### ISSUE TITLE
05 -- Mystery Word -- Adam Hartz

### ISSUE THREAD

**Issue by ahartz1** [tiyd-python-2015-08/assigments#92]
- [x] Blank slate
  - [x] Create a GitHub repo called `mystery-word`
  - [x] Put a `README.md` file in it and copy mystery_word.py and mystery_word_test.py from this repository into it.
- [x] Normal Mode
  - [x] Load words from `/usr/share/dict/words`
  - [x] Write functions to select a subset of the complete word list
  - [x] Write a function to select a word at random from the word list
  - [x] Write a function to display a word with blanks/letters filled in the appropriate spots
  - [x] Write a function to check if a word has been completely guessed
  - [x] Write other helper functions as necessary to help with the flow of the game
  - [x] Run mystery_word_test.py and ensure you pass all the tests
- [ ] Hard Mode (optional)
  - [x] Implement the evil version of the game and put it in evil_mystery_word.py
  - [ ] Write tests and put them in evil_mystery_word_tests.py

**Comment by ahartz1** [comment]
https://github.com/ahartz1/mystery-word

Normal Mode Complete
@jamesmallen @powder-river

**Comment by powder-river** [comment]
@ahartz1 I got one word for you. DAAAAANNNNGGG!!!! Great Job with your code. I like the extra touch you did by making the command line output look very pleasing to the user. I also thought it was a nice touch showing guessed vowels and consonants. My only comment for you right now would be your README. It looks great but remember how we talked about in our workshop how the realm should explain how to run it and what it does? You hit the nail on the head explaining how to run the program, I would recommend describing in more detail what the program does, describe the game. Excellent work Adam this is a strong submission.
